packages/react-core/src/demos/CustomMenus/CustomMenus.md

```diff
@@ -27,78 +27,56 @@ import styles from '@patternfly/react-styles/css/components/Menu/menu';
 
 ## Demos
 
-Custom menus can be constructed using a composable approach by combining the [Menu](/components/menus/menu) and [Menu toggle](/components/menus/menu-toggle) components in unique ways. Composable menus currently require consumer keyboard handling and use of our undocumented [popper.js](https://popper.js.org/) wrapper component called Popper. We understand this is inconvientent boilerplate and these examples will be updated to use [Dropdown](/components/dropdown) in a future release.
+Custom menus can be constructed using a composable approach by combining the [Menu](/components/menus/menu) and [Menu toggle](/components/menus/menu-toggle) components in unique ways. [Dropdown](/components/menus/dropdown), [Select](/components/menus/select), or [MenuContainer](/components/menus/menu#menucontainer) may be used in combination with menu components to handle basic keyboard inputs, or menu components may be connected manually through our undocumented internal [popper.js](https://popper.js.org/) wrapper component called Popper.
```

Comment:
```suggestion
Custom menus can be constructed using a composable approach by combining the [menu](/components/menus/menu) and [menu toggle](/components/menus/menu-toggle) components in unique ways. To handle basic keyboard inputs, [dropdown](/components/menus/dropdown), [select](/components/menus/select), or [`<MenuContainer>`](/components/menus/menu#menucontainer) components may be used in combination with menu components. Additionally, menu components may be connected to each other manually through our undocumented internal [popper.js](https://popper.js.org/) wrapper component called Popper.
```
Is there anything else/more PF specific to link to for Popper? Or is the popper.js site enough for users to be able to piece together that approach?

Reply:
I'm not sure there's a more specific link we can use, short of linking directly to our github code maybe?